langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -423,3 +423,46 @@ public isolated function mergeJson(json j1, json j2) returns json|error = @java:
     'class: "org.ballerinalang.langlib.value.MergeJson",
     name: "mergeJson"
 } external;
+
+# Returns the number of arguments.
+# 
+# ```ballerina
+# value:count(1, 2, 3) ⇒ 3
+# ```
+# 
+# + vs - arguments that are counted
+# + return - number of arguments
+public isolated function count(any|error... vs) returns int = @java:Method {
+    'class: "org.ballerinalang.langlib.value.Count",
+    name: "count"
+} external;
+
+@typeParam
+type Type any|error;
+
+# Returns the first argument.
+#
+# ```ballerina
+# value:first(1, 2, 3) ⇒ 1
+# ```
+# 
+# + v - first argument
+# + vs - rest of the arguments
+# + return - first argument
+public isolated function first(Type v, (any|error)... vs) returns Type {
```

Comment:
We need to return () when there are no arguments in the min/amx/last/first/avg right?, Shall we add tests for those cases as well?

Reply:
Please check this comment. https://github.com/ballerina-platform/ballerina-spec/issues/1135#issuecomment-1200665146

Among `last/first/count`, only the `count` can have no args. In that case, also we have to return 0. Check the test [case](https://github.com/ballerina-platform/ballerina-lang/pull/39667/files#diff-875e3c55bfca3d68e3f568d1ba716a410110ba66aa4fa71a5e1d8fc2f019e032R3067). Therefore There is not a case of returning `()`.

Reply:
This comment was posted after that: https://github.com/ballerina-platform/ballerina-spec/issues/1135#issuecomment-1200678526